tree/ntuple/v7/src/RNTupleDescriptor.cxx

```diff
@@ -1046,6 +1067,46 @@ ROOT::Experimental::Internal::RNTupleDescriptorBuilder::AddExtraTypeInfo(RExtraT
    return RResult<void>::Success();
 }
 
+ROOT::Experimental::Internal::RNTupleSerializer::StreamerInfoMap_t
```

Comment:
Should we add a doc string?

Reply:
There is a (brief) doxygen comment on this method in the header.